### PR TITLE
Removing extra + from ascii that was leaking on the page

### PR DESCRIPTION
**Pull request by lidiazuin**
And also a reminiscent change on the title casing as per https://github.com/neo4j/driver-documentation/pull/302

### CODE REVIEW

**Comment by recrwplay** [comment]
I've rebased this with the Antora migration so it should be good to merge now

**Comment by davidoliverSP2** [comment]
The `+` was being used to try and get the content to indent, so it would be associated to the section above the note. It's possible that the way to get this to work is not by removing the `+`'s, but by experimenting with the line that they appear on. I can probably help with this 😄

File: common-content/modules/ROOT/partials/session-api.adoc
Comment (lidiazuin) [inline common-content/modules/ROOT/partials/session-api.adoc]: So to keep the `+`, would something like this work @davidoliverSP2 ?

```suggestion
+ See link:/docs/operations-manual/4.4/manage-databases/introduction#manage-databases-default[Operations Manual -> The default database] for more information on databases.
```

Comment (davidoliverSP2) [reply common-content/modules/ROOT/partials/session-api.adoc]: Not quite.... adding the `+` in that position would not mean anything to the asciidoc processor, so the + would be added to the start of your line. The `+` must be on it's own line, in the location that the processor will recognise as a continuation of the previous style.

I've taken a closer look (with some trial/error!) and I think the syntax that would create the effect we're after (whereby all the content under the *Database* definition is indented), is this: 

```
# tag::Database[]

Database::
The database with which the session will interact.
When working with a database which is not the default (i.e. the `system` database or another database in Neo4j 4.0 Enterprise Edition), you can explicitly configure the database which the driver is executing transactions against.
+
[WARNING]
====
The resolution of database aliases occurs at connection creation time, which is not under the control of the session.
It is therefore not recommended to alter database aliases while there are live sessions.
The per-user home database is resolved at session creation and when first impersonating a user.
Therefore, the creation of a new session is necessary to reflect a changed home database.
====
+
See link:/docs/operations-manual/4.4/manage-databases/introduction#manage-databases-default[Operations Manual -> The default database] for more information on databases.
+
See below and link:/docs/cypher-manual/4.4/access-control/dbms-administration#access-control-dbms-administration-impersonation[Cypher Manual -> The `DBMS IMPERSONATE` privileges] for more information on impersonation.
+
*Default:* the default database as configured on the server.
+
# end::Database[]
```

This outputs the content, for all manuals, like this:
<kbd>
Screenshot 2022-09-04 at 21 29 59
</kbd>

Comment (lidiazuin) [reply common-content/modules/ROOT/partials/session-api.adoc]: Right! Thanks for making these tests @davidoliverSP2 . I updated the page now and checked on TC and it is looking fine. I'll be waiting for your approval before merging this PR. :)

File: common-content/modules/ROOT/partials/session-api.adoc
Comment (davidoliverSP2) [inline common-content/modules/ROOT/partials/session-api.adoc]:
```suggestion
# tag::Database[]
```

**Review by davidoliverSP2 — APPROVED** [review]
Thanks @lidiazuin 

One small final thing to fix, an extra `#` seems to have slipped in - I've added a suggestion that can remove it

**Pull request merged**
